docs/source/usersguide/processing.rst

```diff
@@ -4,185 +4,18 @@
---------------
-
-.. image:: ../_images/3dgeomplot.png
-   :height: 200px
-
-See below for a simple example of a plots xml file that demonstrates the
-capabilities of 3D voxel plots.
-
-.. code-block:: xml
-
-        <?xml version="1.0" encoding="UTF-8"?>
-        <plots>
-
-          <plot id="1" type="voxel" color="mat">
-              <filename> myplot </filename>
-              <origin> 0 0 0 </origin>
-              <width> 10 10 10 </width>
-              <pixels> 500 500 500 </pixels>
-          </plot>
-
-        </plots>
-
-Voxel plots are built the same way 2D slice plots are, by determining the cell
-or material id of a particle at the center of each voxel. In this example, the
-space covered is the cube between the points (-5,-5,-5) and (5,5,5), with voxel
-centers 10/500 = 0.02 cm apart. The HDF5 voxel files that are produced do not
-specify any color - instead containing only material or cell ids (material id
-in this example) - and thus the ``background``, ``col_spec``, and ``mask``
-elements are not used. If no cell is found at a voxel center, an id of -1 is
-stored.
-
-The voxel plot data is written to an HDF5 file. The voxel file can subsequently
-be converted into a standard mesh format that can be viewed in ParaView, Visit,
-etc. This typically will compress the size of the file significantly. The
-provided utility openmc-voxel-to-silovtk accomplishes this for SILO:
-
-.. code-block:: sh
-
-    openmc-voxel-to-silovtk myplot.voxel -o output.silo
-
-and VTK file formats:
-
-.. code-block:: sh
-
-    openmc-voxel-to-silovtk myplot.voxel --vtk -o output.vti
-
-To use this utility you need either
-
-* `Silomesh <https://github.com/nhorelik/silomesh>`_
-
-or
-
-* `VTK <http://www.vtk.org/>`_ with python bindings. On debian derivatives,
-  these are easily obtained with ``sudo apt-get install python-vtk``
-
-For the HDF5 file structure, see :ref:`io_voxel`.
-
-Once processed into a standard 3D file format, colors and masks can be defined
-using the stored id numbers to better explore the geometry. The process for
-doing this will depend on the 3D viewer, but should be straightforward.
-
-.. image:: ../_images/3dba.png
-   :height: 200px
-
-.. note:: 3D voxel plotting can be very computer intensive for the viewing
-          program (Visit, ParaView, etc.) if the number of voxels is large (>10
-          million or so).  Thus if you want an accurate picture that renders
-          smoothly, consider using only one voxel in a certain direction. For
-          instance, the 3D pin lattice figure at the beginning of this section
-          was generated with a 500x500x1 voxel mesh, which allows for resolution
-          of the cylinders without wasting too many voxels on the axial
-          dimension.
-
-
--------------------
-Tally Visualization
--------------------
+-------------------------
+Working with State Points
+-------------------------
 
 Tally results are saved in both a text file (tallies.out) as well as an HDF5
```

Comment:
I was debating that. I might beef up this section at some point, but for now at least the notebooks demonstrate its usage pretty well.